src/routes/practitoner/practitionerSearch/PractitionerSearch.tsx

```diff
@@ -0,0 +1,153 @@
+  },
+];
+
+const PageWrapper = styled.div`
+  display: flex;
+  padding: var(--spacing-md);
+`;
+
+const ProductsWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-left: var(--spacing-md);
+  width: 100%;
+`;
+
+const ProductsPage = () => {
+  const [selectedFilters, setSelectedFilters] = useState<Array<string> | null>(
+    null
+  );
+  const [productFilterApiParams, setProductFilterApiParams] = useState({});
+  const [isMobileDrawOpen, setIsMobileDrawOpen] = useState(false);
+  const [screenSize, setScreenSize] = useState<WindowSizeEnum>(
+    window.innerWidth > Breakpoints.md
+      ? WindowSizeEnum.LARGE
+      : WindowSizeEnum.SMALL
+  );
+
+  const updateFilters = (newFilter: string) => {
+    setSelectedFilters([...(selectedFilters || []), newFilter]);
+  };
+
+  const clearFilters = () => {
+    setSelectedFilters(null);
+    setProductFilterApiParams({});
+  };
+
+  const toggleMobileFilters = () => {
+    setIsMobileDrawOpen(!isMobileDrawOpen);
+  };
+
+  const setSize = useCallback((dimensions: Dimensions) => {
+    if (dimensions.width > Breakpoints.md) {
+      setScreenSize(WindowSizeEnum.LARGE);
+    } else {
+      setScreenSize(WindowSizeEnum.SMALL);
+    }
+  }, []);
+
+  useWindowResize(setSize);
+
+  const constructApiFilters = (
+    filterName: string,
+    filterValues: Array<MultiSelectOption> | null
+  ) => {
+    const values =
+      filterValues &&
+      filterValues.map((filterValue: MultiSelectOption) => filterValue.name);
+
+    setProductFilterApiParams({
+      ...productFilterApiParams,
+      [filterName]: values,
+    });
+  };
+
+  useEffect(() => {
+    const removeNullValues = removeNullProperties({
+      ...productFilterApiParams,
+    });
+    // Test Purposes
+    console.log(removeNullValues);
+  }, [constructApiFilters]);
+
+  return (
+    <PageWrapper>
+      {screenSize === WindowSizeEnum.LARGE ? (
+        <FilterList
+          setSelectedFilters={updateFilters}
+          selectedFilters={selectedFilters}
+          clearFilters={clearFilters}
+          constructApiFilters={constructApiFilters}
+          filterButtons={FILTER_PRACTITIONER_BUTTON_DATA}
+        />
+      ) : (
+        <>
+          <MenuIcon onClick={toggleMobileFilters} />
+          <MobileFilterList
+            setSelectedFilters={updateFilters}
+            selectedFilters={selectedFilters}
+            isOpen={isMobileDrawOpen}
+            anchorPosition={AnchorPositionEnum.OPEN_LEFT}
+            setIsOpen={toggleMobileFilters}
+            clearFilters={clearFilters}
+            constructApiFilters={constructApiFilters}
+            filterButtons={FILTER_PRACTITIONER_BUTTON_DATA}
+          />
+        </>
+      )}
+      <ProductsWrapper>
+        {FAKE_PRACTITIONER_DATA.map((practitioner) => {
+          return <PractitionerCard practitionerData={practitioner} />;
+        })}
+      </ProductsWrapper>
+    </PageWrapper>
+  );
+};
+
+export default ProductsPage;
```

Comment:
Yeah I don't mind the repetition here tbh. Makes it clearer and there's not loads of repeated logic